Source/WebCore/platform/graphics/NativeImage.h

```diff
@@ -51,7 +51,7 @@ class NativeImage final : public RenderingResource {
 
```

Comment:
I think that keeping this called NativeImage, but not having it inherit from Image will probably continue to be confusing. Perhaps leaning into its "resource"-ness and going with `NativeImageResource`? Anything to avoid the `Image` suffix.

Reply:
Yes I agree. I will address this in a separate patch.